### ISSUE TITLE
Redesign Twitter component 

### ISSUE THREAD

**Issue by joyjamir** [DrewAllen88/Enhancements#42]
Redesign Twitter component so that it does not cause (or require) the locale page to auto-publish.

Lives in the footer of each page:
![image](https://cloud.githubusercontent.com/assets/8258720/8937888/33930756-3529-11e5-9382-8e7076966c1d.png)

**Comment by DrewAllen88** [comment]
BAU item. Reach out to Mike Vossi for further clarification if needed.

**Comment by monalishah** [comment]
It means twitter feed component will be authored on every page where there is a need to display feeds.
Please confirm.

**Comment by DrewAllen88** [comment]
@joyjamir That doesn't sound like an optimal approach.

**Comment by joyjamir** [comment]
No that will not work.  We need the Twitter feed to update without resulting in activation of the locale page, but the solution cannot be that we need to author the Twitter feed on each page of the site.

**Comment by monalishah** [comment]
Yes Drew. You are right. I was bit confused regarding the requirement. Now its cleared.

**Comment by abhishek-a** [comment]
MAR-2292 Jira created for it hence do not update the ticket on HGITHUB anymore. this is shifted to Release 3.
